Code/Framework/AzToolsFramework/AzToolsFramework/ToolsComponents/ScriptEditorComponent.cpp

```diff
@@ -1008,6 +1011,7 @@ namespace AzToolsFramework
                 AzFramework::ScriptComponent::Reflect(context);
 
                 serializeContext->Class<ScriptEditorComponent, EditorComponentBase>()
+                    ->Version(2)
```

Comment:
Do we need to add a custom json serializer to process something differently with this version bump? The version isn't actually serialized anymore, but if this is just for consistency then ignore my comment.